### ISSUE TITLE
Allow a user to retrieve the transparency logs for an artifact using the CLI

### ISSUE THREAD

**Issue by erwin1** [pyrsia/pyrsia#688]
Part 1: define what a uses wants to see and implement in a mocked way in the CLI

(Arrow 6b - https://pyrsia.io/docs/developers/pyrsia-high-level-overview/)

Since for searching the logs we need information from the docker manifest - it might make sense to duplicate/copy the manifest into the transparency log.

**Comment by betarelease** [comment]
Depends on #1399

**Comment by dgarus** [comment]
@erwin1 @betarelease

**Comment by betarelease** [comment]
duplicate of #1399. Closing.
